### DISCUSSION TITLE
RelationManager managed as a resource

### DISCUSSION THREAD

**Discussion by alepieser** [filamentphp/filament#6193] [Help]
Hey !

I would know if it's possible to manage a relation as we could with a resource i.e. to have list in a dedicated page instead of edit page of parent

**Comment by danharrin** [comment]
https://filamentphp.com/tricks/relation-manager-edit-record-without-modal

**Reply by alepieser** [reply]
Thanks @danharrin for your quick answer.

Sorry I was not very clear in my question. 

I have a resource Pool who can have many Vouchers. 
On Pool's table, I would like add a table's action to go to a page where I can see all the vouchers available for the pool selected.

So instead of to see the list of vouchers associated to a pool in the pool's edit page (relation manager tabs), I would to like to have a dedicated page where I can access directly from the pool table.

**Reply by danharrin** [reply]
You would need a custom page, but I don't really understand why you would want to see a table of vouchers in two different places

**Reply by alepieser** [reply]
The idea is to access directly to the management of vouchers available for a pool without pass by the edition of the pool because the form for the pool is complex & heavy.

It was a way to simplify the thing. A page to edit the pool configuration, a page to manage the vouchers associated

**Reply by danharrin** [reply]
You would need a custom page that uses the table builder
